src/data/botData.ts

```diff
@@ -0,0 +1,212 @@
+import getDatabase from ".";
+import { randomBytes } from "crypto";
+import { DatabaseSession, Databases, Session } from "./types";
+
+const db = getDatabase(Databases.BOT_DATA)
+
+// ------------- SESSIONS -------------
+
+/**
+ * Randomly generates a session token.
+ * 
+ * @param [bytes=36] The number of bytes to generate.
+ * @returns A randomly generated session token.
+ */
+function generateSessionToken(
+    bytes: number = 36
+): string {
+    return randomBytes(bytes).toString('base64')
```

Comment:
So this is not only randomly generating a session token but encrypting it in base64?

Reply:
It uses the crypto.js library to generate 36 random bytes and convert them into a string encoded with base64.

Reply:
It uses the crypto.js library to generate 36 random bytes and convert them into a string encoded with base64.